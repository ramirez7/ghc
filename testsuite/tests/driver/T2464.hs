{-# LANGUAGE CPP #-}
{-# OPTIONS_GHC -DTEST #-}
#ifdef TEST
{-# LANGUAGE DeriveFunctor #-}
#endif
{-# LANGUAGE NullaryTypeClasses #-}  -- generate a deprecation warning

module Test2464 where
data T a deriving (Functor)
