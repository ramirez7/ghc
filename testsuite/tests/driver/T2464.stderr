
T2464.hs:6:14: warning:
    -XNullaryTypeClasses is deprecated: use -XMultiParamTypeClasses or pragma {-# LANGUAGE MultiParamTypeClasses #-} instead
